### PR TITLE
new clickable card that behaves like a button

### PR DESCRIPTION
**Pull request by edewit**
fixes: #3214

## Checklist:

- [ ] Code has been tested locally by PR requester
- [ ] User-visible strings are using the react-i18next framework (useTranslation)
- [ ] Help has been implemented
- [ ] axe report has been run and resulting a11y issues have been resolved
- [ ] Unit tests have been created/updated

### CODE REVIEW

**Review by jonkoops — APPROVED** [review]

File: apps/admin-ui/src/user-federation/UserFederationSection.tsx [hunk 1]
```diff
@@ -194,11 +194,8 @@ export default function UserFederationSection() {
...
             <Gallery hasGutter>
               {providers.map((p) => (
-                <Card
+                <ClickableCard
                   key={p.id}
-                  className="keycloak-empty-state-card"
-                  role="button"
-                  isHoverable
                   onClick={() =>
                     navigate(toProvider({ realm, providerId: p.id! }))
```
Comment (jonkoops) [inline apps/admin-ui/src/user-federation/UserFederationSection.tsx:200]: Since a click causes a navigation it would be great if this could be a link.

**Review by agagancarczyk — APPROVED** [review]

**Comment by agagancarczyk** [comment]
> also fixes: #3215

This is great @edewit I have also noticed that the aria-labels for Bitbucket, Instagram, Microsoft and Paypal in FontAwesomeIcon.tsx are inconsistent with the rest of the icons. For example, if you tab into Bitbucket, the screen reader is reading `Bitbucket icon Bitbucket Button Not Pressed`, but for Facebook, it's reading `Facebook Button Not Pressed`. 

In short it would be great to keep all aria-labels consistent. I'd suggest removing world `icon` from Bitbucket, Instagram, Microsoft and Paypal icons defined in FontAwesomeIcon.

Would you be able to do it in this PR or would you like me to do it in a separate PR once this PR is merged? 

Thanks!

**Comment by edewit** [comment]
@agagancarczyk removed the aria labels from the `FontAwesomeIcon` components as they are the same as the button text

**Comment by agagancarczyk** [comment]
> @agagancarczyk removed the aria labels from the `FontAwesomeIcon` components as they are the same as the button text

Thanks @edewit !

**Review by jonkoops — APPROVED** [review]

**Pull request merged**